types/react-native-snap-carousel/index.d.ts

```diff
@@ -43,7 +43,7 @@ export interface CarouselProps<T> {
      * Function that takes an item from the `data` array and returns a React
      * Element. See `react-native`'s `FlatList`
      */
-    renderItem(item: { item: T; index: number }, parallaxProps?: AdditionalParallaxProps): React.ReactNode;
+    renderItem(item: { item: T; index: number }, parallaxProps?: AdditionalParallaxProps): React.ReactElement | null;
```

Comment:
This doesn't seem right. Returning `ReactElement | null` should only happen in components; since this is a render prop, it's okay to return `ReactNode`.